src/CattyTests/Formula/FormulaTest.swift

```diff
@@ -0,0 +1,186 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+final class FormulaTest: XCTestCase {
+    var formulaManager: FormulaManagerProtocol!
+    var interpreter: FormulaInterpreterProtocol!
+
+    override func setUp() {
+        let screenSize = Util.screenSize(true)
+        formulaManager = FormulaManager(sceneSize: screenSize)
+        interpreter = formulaManager
+    }
+
+    func testIsSingularNumberFormula() {
+        //var formula = Formula(integer: 1)
+        if let formula = Formula(integer: 1) {
+            XCTAssertTrue(formula.isSingularNumber())
+            print("yes")
```

Comment:
Please remove print message and comment

Reply:
Done. I totally overlooked it. Thank you for the hint!